packages/AlertDialog/index.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {useState} from 'react';
+import Modal from 'react-native-modalbox';
+import styled from '@emotion/native';
+
+import {StyleProp, ViewStyle} from 'react-native';
+
+const ContentContainer = styled.SafeAreaView`
+  flex-grow: 1;
+  flex-shrink: 1;
+  align-self: stretch;
+  justify-content: center;
+  margin: 24px;
+`;
+
+const ButtonContainer = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 32px;
+  margin-bottom: 12px;
+`;
+
+const ModalButton = styled.TouchableOpacity`
+  margin: 0px 8px;
+  width: 80px;
+  height: 36px;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ModalInput = styled.TextInput<ColorProps>`
+  height: 40px;
+  margin-top: 20px;
+  padding: 4px;
+  font-size: 14px;
+  border: 1px solid ${(props) => props.color};
+`;
+
+const ButtonText = styled.Text<ColorProps>`
+  font-size: 14px;
+  font-weight: bold;
+  color: ${(props) => props.color};
+`;
+
+type Styles = {
+  modalContainer?: StyleProp<ViewStyle>;
+};
+
+interface Props {
+  isOpen?: boolean;
+  type?: 'alert' | 'confirm' | 'prompt';
+  color?: string;
+  styles?: Styles;
+  backdropOpacity?: number;
+  backdropPressToClose?: boolean;
+  children?: React.ReactNode;
+  onPress: (result: boolean | string | null) => void;
+}
+
+interface ColorProps {
+  color: string;
+}
```

Comment:
The styling principle in `dooboo-ui` is not to expose each nested style prop and rather expose the style itself. 
For example, `ViewStyle`, `TextStyle`...